homeassistant/components/vicare/__init__.py

```diff
@@ -42,12 +44,22 @@ class HeatingType(enum.Enum):
     fuelcell = "fuelcell"
 
 
+@contextmanager
+def catch_not_supported():
```

Comment:
There is a built-in context manager for this: [`contextlib.suppress`](https://docs.python.org/3/library/contextlib.html#contextlib.suppress)